client/src/pages/TotalExpensesChart.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import { Typography, Divider } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import Paper from "@material-ui/core/Paper";
+import {
+  Chart,
+  ArgumentAxis,
+  ValueAxis,
+  SplineSeries
+} from "@devexpress/dx-react-chart-material-ui";
+
+const useStyles = theme => ({
+  root: {
```

Comment:
doesn't look like you're using this style here

Reply:
I'm working on the styling now! :)